test/clj/rems/api/test_email.clj

```diff
@@ -39,3 +40,42 @@
         (is (response-is-forbidden? response))
         (is (str/includes? (get-in response [:headers "x-rems-roles"])
                            "logged-in"))))))
+
+(deftest test-send-reviewer-reminder
+  (let [api-key "42"
+        user-id "developer"
+        outbox-emails (atom [])
+
+        app-id (test-data/create-application! {:actor "alice"})]
+    (test-data/command! {:type :application.command/submit
+                         :application-id app-id
+                         :actor "alice"})
+    (test-data/command! {:type :application.command/request-comment
```

Comment:
Likewise here, even if request-comment is changed to request-review, it is still difficult to map the assumed semantics of the word to the conceptual difference between request-review and request-decision.

(I want to emphasize that it's by no means a problem introduced by this patch, but this patch kind of builds on the existing problem, so that's why I'm bringing it up here.)

At least, there could be a TODO comment here for changing the name of the command to request-review.

Reply:
There's a comment about it in rems.application.events/CommentedEvent

Reply:
Created #1887